frontend/lambda-in/src/Components/Billing/billing.css

```diff
@@ -1,7 +1,152 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
'CSS Reset', needs to be at the top of the production-CSS file when it is compiled.
A better place to write it is at the top of `index.css`.